db/migrations/20250420234337_moderation_tables.sql

```diff
@@ -0,0 +1,77 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE report_reasons (
+	report_reason_id bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	title text NOT NULL UNIQUE,
+	description text DEFAULT '' NOT NULL
+);
+
+INSERT INTO report_reasons (title, description)
+VALUES
+    ('Hate Speech', 'Content that promotes hatred against protected groups'),
+    ('Harassment', 'Content that harasses, intimidates, or bullies others'),
+    ('Misinformation', 'Deliberately false or misleading content'),
+    ('Illegal Content', 'Content that violates local, state, or federal laws'),
+    ('Spam', 'Unsolicited commercial or repetitive content'),
+    ('Violence', 'Content that promotes or glorifies violence'),
+    ('Copyright Violation', 'Content that infringes on copyrighted material'),
+    ('Personal Information', 'Sharing of private personal information without consent'),
+    ('Other', 'Violation not covered by other categories');
+
+CREATE TABLE reports (
+    report_id bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    reported_by_profile_id bigint NOT NULL,
+    report_reason_id bigint NOT NULL,
+    report_reason_extra text DEFAULT '' NOT NULL,
+    created timestamp without time zone NOT NULL
+);
+
+ALTER TABLE reports ADD CONSTRAINT reports_report_reason_id_fkey FOREIGN KEY (report_reason_id) REFERENCES report_reasons(report_reason_id);
+ALTER TABLE reports ADD CONSTRAINT reports_reported_by_profile_id_fkey FOREIGN KEY (reported_by_profile_id) REFERENCES profiles(profile_id);
+
+CREATE TABLE actions (
+	action_id bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
same as reports - is `bigint` overkill?

Reply:
Totally is, but again, I was weighing up maintaining convention vs appropriately sized column sizes and I sided with convention. I'm not set on the idea though

Reply:
let's stay consistent with the other tables